docs/pyodide/afterBody.js

```diff
@@ -0,0 +1,77 @@
+const initialSketch = checkForSketch();
```

Comment:
Nice touch on moving this to a separated js file.

Reply:
Since we're using jQuery here, shouldn't we wrap this whole Javascript inside of a `$(document).ready()` callback?

Reply:
Going from what I can see in [view_sketch.html](https://github.com/berinhard/pyp5js/blob/develop/pyp5js/http/templates/view_sketch.html) the editor config should be outside of this call. 
Either way, I'll send a commit changing this, as most of the code in afterBody should be included in this call.

Reply:
Done in ba89c38 but I used the vanilla JS "DOMContentLoaded" which should have the same effect.